bin/release

```diff
@@ -317,17 +317,20 @@ def add_libs(rel_name, galette_archive):
     p1 = subprocess.Popen(npm_cmd, shell=True, cwd=npm_dir)
     p1.wait()
 
-    wp_cmd = 'npm run-script build-dist'
+    wp_cmd = 'gulp'
```

Comment:
When running the release script, I got the following error:
```
/bin/sh: ligne 1: gulp : commande introuvable
```

I do not understand if there is some dependency missing or else; trying to run `npm run-script gulp` (the command used for galette dependencies install fails with the same error message.

Reply:
I think I got it... In package.json file, the script name for the gulp command is `build`. 
```suggestion
    wp_cmd = 'build'
```